go-controller/pkg/util/ovs.go

```diff
@@ -815,19 +820,34 @@ type NBTxn struct {
 	args    []string
 	txnArgs []string
 	env     []string
+	maxArgs int
 }
 
 // NewNBTxn returns a new ovn-nbctl transaction request object
 func NewNBTxn() *NBTxn {
 	args, env := getNbctlArgsAndEnv(ovsCommandTimeout, []string{}...)
+
+	var maxArgs int
+	out, err := exec.Command("getconf", "ARG_MAX").Output()
```

Comment:
@trozet let's just do this once in init() and cache the value?

Reply:
ok